BigEval.js

```diff
@@ -570,6 +570,12 @@ BigEval.prototype.compile = function (expression) {
 			((i > 1 && tokens[i - 2].type === TokenType.OP) || i === 1)
 			) {
 
+			console.log(tokens[i - 2]);
+
+			if (i > 1 && tokens[i - 2].value==='!') {				
+				continue
+			}
+
```

Comment:
Adding a test for this fix would be nice too. We have automated tests if you pull the latest commits from `master`.

Can you pull the latest commits from `master` and add a test or two? Thanks.

Reply:
Have added a test.